keps/sig-node/4004-deprecate-kube-proxy-version/README.md

```diff
@@ -0,0 +1,261 @@
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (Optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Deprecate the `status.nodeInfo.kubeProxyVersion` field of v1.Node
+
+## Motivation
+
+This field is not accurate, the field is set by kubelet, which does not actually know the kube-proxy version
+
+### Goals
+
+Deprecate the `status.nodeInfo.kubeProxyVersion` field and then do the hard work of finding all the places using it and getting them off in time.
+
+### Non-Goals
+
+## Proposal
+
+- Deprecate the `status.nodeInfo.kubeProxyVersion` field of v1.Node
+- Remove the gcp cloud provider check for this field
```

Comment:
ah, yeah, I feel like this is more "Goals" than "Proposal"...

Also, wherever it is that you mention the GCP cloud provider, you need to be clear that that's just one specific example, and that we may find others.

(Also, maybe link to the PRs I already filed: https://github.com/kubernetes/cloud-provider-gcp/pull/533, https://github.com/kubernetes/kubernetes/pull/117806.)

Reply:
> ah, yeah, I feel like this is more "Goals" than "Proposal"...
> 
> Also, wherever it is that you mention the GCP cloud provider, you need to be clear that that's just one specific example, and that we may find others.
> 
> (Also, maybe link to the PRs I already filed: [kubernetes/cloud-provider-gcp#533](https://github.com/kubernetes/cloud-provider-gcp/pull/533), [kubernetes/kubernetes#117806](https://github.com/kubernetes/kubernetes/pull/117806).)

I have made some changes, please see if it is more appropriate